a2sv_community_portal_mobile/lib/features/application_page/presentation/screen/home.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:a2sv_community_portal_mobile/core/utils/media_query.dart';
+import 'package:a2sv_community_portal_mobile/features/application_page/presentation/widgets/custom_button.dart';
+import 'package:a2sv_community_portal_mobile/features/application_page/presentation/widgets/custom_card.dart';
+import 'package:a2sv_community_portal_mobile/features/application_page/presentation/widgets/stepper_container.dart';
+import 'package:a2sv_community_portal_mobile/features/application_page/presentation/widgets/upper_bar.dart';
+import 'package:flutter/material.dart';
+// Remove this unused import
+
+class Home extends StatelessWidget {
+  const Home({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: const UpperBar(),
+      backgroundColor: Colors.white,
+      body: Column(
+        children: [
+          const Padding(
+            padding: EdgeInsets.only(top: 17, bottom: 17.0),
+            child: StepperBox(
+              text: '1 of 4',
+              percentage: 0.20,
+              currentStep: 'stepName',
+              nextStep: 'nextStepName',
+            ),
+          ),
+          Expanded(
+            child: Center(
+              child: SingleChildScrollView(
+                child: Column(
+                  children: const [
+                    CustomCard(
+                      title: 'subStepName',
+                      description:
+                          'Lorem ipsum dolor sit alflkafl alflakfkl aflflka aflfkasf kaflka amet consectetur. ',
+                      done: false,
+                    ),
+                    CustomCard(
+                      title: 'subStepName',
+                      description:
+                          'Lorem ipsum dolor sit alflkafl alflakfkl aflflka aflfkasf kaflka amet consectetur. ',
+                      done: true,
+                    ),
+                    CustomCard(
+                      title: 'subStepName',
+                      description:
+                          'Lorem ipsum dolor sit alflkafl alflakfkl aflflka aflfkasf kaflka amet consectetur. ',
+                      done: true,
+                    ),
+                    CustomCard(
+                      title: 'subStepName',
+                      description:
+                          'Lorem ipsum dolor sit alflkafl alflakfkl aflflka aflfkasf kaflka amet consectetur. ',
+                      done: true,
+                    ),
+                    CustomCard(
+                      title: 'subStepName',
+                      description:
+                          'Lorem ipsum dolor sit alflkafl alflakfkl aflflka aflfkasf kaflka amet consectetur. ',
+                      done: true,
+                    ),
```

Comment:
it would be clean if you made this a loop or a map

Reply:
it is for sample I will do it in the dependency injection